upgrade-instructions.md

```diff
@@ -19,7 +19,22 @@ Change list:
  - [Drop Cassandra schema version prior version 8](#drop-cassandra-schema-version-prior-version-8)
  - [Adopt UnboundID as a LDAP library](#adopt-unboundid-as-a-ldap-library)
  - [Review the architecture of the RabbitMQ event bus](#review-the-architecture-of-the-rabbitmq-event-bus)
+ - [Cassandra Schema update to V11](#cassandra-schema-update-to-v11)
  
+### Cassandra Schema update to V11
+
+Date 23/06/2021
+
+JIRA: https://issues.apache.org/jira/browse/JAMES-3516
+
+Concerned product: Distributed James
+
+Add threadId column to messageIdTable and imapUidTable in order to get a message's threadId.
+
+In order to benefit from this work, you need to [upgrade to the latest schema version](https://github.com/apache/james-project/blob/master/src/site/markdown/server/manage-webadmin.md#upgrading-to-the-latest-version).
+
+A James restart is advised after this migration in order to skip schema version reads.
```

Comment:
Nope.

Reply:
It's more the other way around right? You should update the cassandra tables first before upgrading James?

Reply:
Yes.